src/common.rs

```diff
@@ -695,6 +687,18 @@ impl Info<'_> {
 }
 
 impl BytesPerPixel {
+    pub(crate) fn for_prediction(bpp: usize) -> Self {
```

Comment:
Could you call this `from_usize` or similar?

Reply:
Done.  I assume that you plan to squash the last 2 small commits (`cargo fmt` + this rename) when merging?